CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## Changelog
 
+### 0.10.2
+
+**Commit Delta**: [Change from 0.10.1 release](https://github.com/plus3it/watchmaker/compare/0.10.1...0.10.2)
+
+**Released**: 2018.09.27
+
+**Summary**:
+
+*   Adds a gitlab-ci pages config to build Watchmaker docs
+*   Uses new hosting location to retrieve Salt packages
+*   Restricts click version on py2.6
+*   ash-linux-formula
+    -   Pins docker images to digest for updating by dependabot
```

Comment:
This is only relevant to the tests, there's no functional changes for the user. We don't need a changelog entry for this type of change.

Reply:
Okay.  I will remove it.